toolbox/ci2gubernator/lib/ci2gubernator.go

```diff
@@ -57,10 +57,10 @@ type Converter struct {
 }
 
 // NewConverter creates a Converter
-func NewConverter(bucket, org, repo, job string, build int) *Converter {
+func NewConverter(bucket, org, repo, job, mux string, build int) *Converter {
```

Comment:
why is "mux"?

Reply:
MUX is short for multiplexer but I changed to `stage` to be consistent with the main file and to be more self-explanatory.

Reply:
Have you not pushed the commit for this change?